THANKS.txt

```diff
@@ -49,6 +49,7 @@ Alan McIntyre for updating the NumPy test framework to use nose, improve
 Joe Harrington for administering the 2008 Documentation Sprint.
 Mark Wiebe for the new NumPy iterator, the float16 data type, improved
     low-level data type operations, and other NumPy core improvements.
+Kernc for scale.
```

Comment:
There's not, the contributor guidelines suggest you add yourself. Full name would be better than GitHub handle.

We should probably change that file in some way or get rid of it completely. We acknowledge and thank all contributors in the release notes; this file is often not updated; when you do update it in a PR there's often a merge conflict.